pom.xml

```diff
@@ -54,6 +57,28 @@
             <artifactId>commons-codec</artifactId>
             <version>1.11</version>
         </dependency>
+        <!--logging-->
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>${logback-json-classic.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>${logback-json-classic.version}</version>
+        </dependency>
```

Comment:
`logback-core` should be a transitive dep to `logback-classic`. So this is not needed.

Reply:
no, need to define this dependency,otherwise we get this Warning:

`09:36:02,520 |-WARN in net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder@1ddf84b8 - Logback version is prior to 1.2.0.  Enabling backwards compatible encoding.  Logback 1.2.1 or greater i`